lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -4104,3 +4104,21 @@ mapping:
         per_host: true
         desc: |
           Enable the integration of the Galaxy Help Forum in the tool panel. This requires the help_forum_api_url to be set.
+
+      file_source_temp_dir:
+        type: str
+        required: false
+        desc: |
+          Directory to store temporary files for file sources. This defaults to new_file_path if not set.
+      
+      file_source_webdav_use_temp_files:
+        type: bool
+        default: false
```

Comment:
Why is the default false ? Could we not hardcode this to true ? When would you ever want this to be false ? I feel like we should just patch up quirks in file sources, not burden admins with this stuff.

Reply:
I agree, I missed that default to false. It should be true.

Reply:
I will switch the default to ``True`` - that was an oversight, the actual underlying plugin defaults to ``False`` (https://github.com/PyFilesystem/webdavfs/blob/master/webdavfs/webdavfs.py#L167) - so it seems like it should be configurable to me but I will drop it if y'all insist.

Reply:
Ahh I see, that is because temp_path is also None by default at the PyFileSystem plugin level, but in our file source plugin we probably can fix it always to be true, since we have (now) the temp_path set in the config or the fallback in case of None.